libraries/AP_Scripting/applets/VTOL-quicktune.lua

```diff
@@ -283,8 +322,8 @@ end
 
 -- return gain limits on a parameter, or 0 for no limit
 function gain_limit(pname)
-   if string.sub(pname, 1, 3) == "YAW" then
-      local suffix = string.sub(pname, -2)
+   if param_axis(pname) == "YAW" then
+      local suffix = param_suffix(pname, -2)
```

Comment:
What does the extra argument `-2` do? 

```suggestion
      local suffix = param_suffix(pname, -2)
```

Reply:
fixed, thanks